bin/run_docker.py

```diff
@@ -323,6 +326,27 @@ def run_docker(
         (key for key in volumes.keys() if "output" in volumes[key]["bind"]), None
     )
 
+    # If the submission is not a Docker image, create an invalid output file
+    # store the error message in a log file, and end the script call
+    if "InputError" in docker_image:
```

Comment:
Nit: One thought of this is that we used to have a "validate docker submission" step that does the validation, this way it keeps this step clean

Reply:
Good call - can do